### ISSUE TITLE
*Datatable* - Configure Columns - "We can't launch this flow because of a variable error" Can't configure columns.

### ISSUE THREAD

**Issue by Mr-Jree** [alexed1/LightningFlowComponents#716]
### Steps to reproduce

1. Click on Configure Column after **already completing the column wizard once**
2. [We can't launch this flow because of a variable error.](https://imgur.com/WdwmdPu)
3. Cannot configure columns anymore after the first time.

### Expected behaviour
What should happen is this error shouldn't appear when trying to configure a column past the first time.

### Actual behaviour
Configure column action bugs and I have to remake the entire datatable if I wanted to change the columns.

### Screenshots
https://imgur.com/WdwmdPu

**Comment by ericrsmith35** [comment]
Please review the troubleshooting tips here and see if any apply.
https://unofficialsf.com/datatable-lightning-web-component-for-flow-screens-2/#Restrictions

If you are not a System Admin, make sure you are assigned the Permission Set that is included with the package.
Confirm that the current versions of the Flow Base Packs are successfully installed.  (https://unofficialsf.com/flow-action-and-screen-component-basepacks/)

**Comment by Mr-Jree** [comment]
Hi Eric,

1) Clickjack protection for VF pages with headers was already unchecked
2) Debug and Development mode were both already unchecked
3) I am a system admin.

**Comment by ericrsmith35** [comment]
Can you confirm the versions of the installed Base Packs?

Here are a few Salesforce bugs that can cause issues:

1. Including a "section" component on your flow screen
2. Trying to save an edited record with a picklist field
3. Trying to update a record with a non custom defined integer field
4. Having any record in your input collection with a value in a geo-location field

**Comment by Mr-Jree** [comment]
1. 2.17
2. Nope, not doing that here
3. What is a non-custom defined integer field? That's not happening though, just trying to configure columns from a get record element in screen flow
4. No geo-location fields in any records in the input collection.

**Comment by ericrsmith35** [comment]
What version of the FlowScreenComponentsBasePack is installed?

**Comment by ericrsmith35** [comment]
Yes, you have the current version of FlowActionBasePack with 2.17.

You need the current version of FlowScreenComponentsBasePack (2.2.1) to be installed as well.

**Comment by Mr-Jree** [comment]
I have the 2.2.1 of the FlowScreenComponentsBasePack installed!

Sorry, thought you meant what FlowActionBasePack version I had.

**Comment by rcurl83** [comment]
I am getting the same error when trying to show a list of tasks. What I have found is it when I included the subject field from a task, then I cannot launch the configure columns the second time.  I have updated to all the latest package versions in my sandbox.
